lib/java/com/google/android/material/internal/CollapsingTextHelper.java

```diff
@@ -668,15 +768,8 @@ private void calculateUsingTextSize(final float textSize) {
       // collapsed text size
       final float scaledDownWidth = expandedWidth * textSizeRatio;
```

Comment:
I think this variable is no longer read?

Reply:
Oh, right. I removed it in a0cca8a.